test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -229,10 +230,8 @@ func (r *DockerMachinePoolReconciler) reconcileDelete(ctx context.Context, clust
 		}
 
 		if len(errs) > 0 {
-			return ctrl.Result{}, kerrors.NewAggregate(errs)
+			return kerrors.NewAggregate(errs)
 		}
```

Comment:
```suggestion
		}
		return nil
```

I think we still have to return

Reply:
fixed